lib/global_uid/active_record_extension.rb

```diff
@@ -35,7 +35,7 @@ def generate_uid
 
       def generate_many_uids(count)
         GlobalUid::Base.with_servers do |server|
-          return server.allocate(self, count: count)
+          return Array(server.allocate(self, count: count))
```

Comment:
I thought maybe it would be better in `Allocator#allocate_many` to call `#allocate_one if count == 1`. But then I noticed we do a similar switching in `Server#allocate` so meh… I end up thinking the suggested change is fine.